repair_tests/repair_test.py

```diff
@@ -1271,6 +1271,13 @@ def _test_failure_during_repair(self, phase, initiator=False):
             "org.apache.cassandra.repair.SomeRepairFailedException: null"
         ]
 
+        if cluster.version() < '4.0':
+            # CASSANDRA-17466 was not back ported to 3.x, so ignore the error
+            for a in ['java.lang.InterruptedException',
+                      ('java.util.concurrent.CancellationException: '
+                       'Task was cancelled')]:
+                self.fixture_dtest_setup.ignore_log_patterns.append(a)
+
```

Comment:
Why it was not ported and we ignore it? It is a bug

Reply:
Question for @maedhroz, but I think I would answer as the following: repair failed and we have a race condition where we generate an ugly log, the expected behavior of repair isn't impacted, this is a logging issue mostly I believe...

@maedhroz this didn't close the SSTables right?  

If we do wish to backport that ticket, I request we do it in another JIRA

Reply:
It's absolutely a bug, but I don't think the burden should be on @dcapwell to fix my lack of a 3.x backport to get this work merged. +1 on a new Jira.

Reply:
reverted